lib/recurly/usage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class Recurly_Usage extends Recurly_Resource
+{
+  var $subUuid;
+  var $addOnCode;
+
+  protected static $_writeableAttributes;
+
+  public static function init()
+  {
+    Recurly_Usage::$_writeableAttributes = array(
+      'amount','merchant_tag','usage_type', 'unit_amount_in_cents',
```

Comment:
total nit pick but can we either have spaces after all the commas or none of them?